steampy/client.py

```diff
@@ -311,10 +316,77 @@ def fetch_price(self, item_hash_name: str, game: GameOptions, currency: str = Cu
             raise TooManyRequests("You can fetch maximum 20 prices in 60s period")
         return response.json()
 
+    @login_required
+    def get_my_market_listings(self) -> dict:
+        response = self._session.get("%s/market" % SteamClient.COMMUNITY_URL)
+        if response.status_code == 200:
+            assets_descriptions = json.loads(text_between(response.text, "var g_rgAssets = ", ";"))
+            listing_id_to_assets_address = get_listing_id_to_assets_address_from_html(response.text)
+            listings = get_market_listings_from_html(response.text)
+            listings = merge_items_with_descriptions_from_listing(listings, listing_id_to_assets_address,
+                                                                  assets_descriptions)
+            return listings
+        else:
+            raise FetchException("Steam market page return a %s" % response.status_code)
+
```

Comment:
Ok, I have added an Exception just for that because I think is wrong to return an empty dictionary (it return an empty dict if nothing is found on the html because I could have 0 listings) if the http code isn't 200,